README.md

```diff
@@ -74,10 +74,12 @@ Links marked with *WIP* are Work In Progress and still in development - don't ex
 ## Libraries
 
 - [gba++](https://github.com/felixjones/gbaplusplus) - WIP modern C++ header-only library for GBA.
+- [agbabi](https://github.com/felixjones/agbabi) - System functions (memory, division, interrupts, coroutines)
```

Comment:
This project looks WIP and doesn't have a README at all, should be removed for now (you can open an Issue mentioning it so we can keep track of the documentation progress and the project status)

Reply:
Done! It's an exceptionally useful lib (even for the ucontext implementation alone), but doesn't sell itself very well. Let's be sure to add it once there's a readme and an indication of how complete it is :smile: